core/src/main/kotlin/org/evomaster/core/problem/httpws/service/HttpWsFitness.kt

```diff
@@ -292,7 +292,7 @@ abstract class HttpWsFitness<T>: FitnessFunction<T>() where T : Individual {
             }
 
             val response = try {
-                client.target(baseUrl + cl.loginEndpointUrl)
+                client.target(if (cl.fullUrl) cl.loginEndpointUrl else (baseUrl + cl.loginEndpointUrl))
```

Comment:
when the loginUrl is relative, should have a check if it starts with "/".
also, need to check if `baseUrl` is not ending already with "/" when concatenating the two